debian/changelog

```diff
@@ -0,0 +1,73 @@
+mcrouter (0.22.0-0reddit1ubuntu6) trusty; urgency=medium
```

Comment:
Since none of these were released, could you just squash it down into one release?